webpack/assets/javascripts/react_app/components/charts/ChartHeader.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Component from '../common/BindingComponent';
+
+export default class extends Component {
+  constructor(props) {
+    super(props);
+    this.bind(
+      'onClick'
+    );
+  }
+
+  onClick() {
+    this.props.onClick();
+  }
+
+  render() {
+    // TODO: refactor to handle tooltip and placement here or in chart box
```

Comment:
Tried that, thought it was unduly complex with no benefit here. Decided that the regular bootstrap tooltip would be better. TODO is stale and is deleted. You'll see it in next push.